src/patternfly/components/Button/button.scss

```diff
@@ -143,6 +143,12 @@
   --#{$button}--m-plain--active--Color: var(--#{$pf-global}--Color--100);
   --#{$button}--m-plain--disabled--Color: var(--#{$pf-global}--disabled-color--200);
   --#{$button}--m-plain--disabled--BackgroundColor: transparent;
+  --#{$button}--m-plain--m-inline--FontSize: inherit;
+  --#{$button}--m-plain--m-inline--PaddingTop: 0;
+  --#{$button}--m-plain--m-inline--PaddingRight: var(--#{$pf-global}--spacer--sm);;
+  --#{$button}--m-plain--m-inline--PaddingBottom: 0;
+  --#{$button}--m-plain--m-inline--PaddingLeft: var(--#{$pf-global}--spacer--sm);;
+  --#{$button}--m-plain--m-inline--LineHeight: normal;
```

Comment:
Updating from `normal` to `var(--#{$pf-global}--LineHeight--md)` causes a slight difference in height between the button and whatever text it might be next to, with the button height slightly exceeding the height of the text next to it.

With the global var:

![Text height next to plain inline button with global var](https://github.com/patternfly/patternfly/assets/70952936/0f3bc61b-b5ae-4bd1-a3dc-db04ebbba050)
![Plain inline button height with global var](https://github.com/patternfly/patternfly/assets/70952936/d5f33810-2291-4364-b85d-e77405c5ed52)


With the `normal` keyword:

![Text height next to plain inline button with normal keyword](https://github.com/patternfly/patternfly/assets/70952936/0ec30cf9-3640-41cb-a648-8a2bb0ba94ef)
![Plain inline button with normal keyword](https://github.com/patternfly/patternfly/assets/70952936/8810a59d-da74-4f69-926e-eaabe4189c70)

Screenshots are from Firefox but Chrome has similar height differences, which get more significant as the font size increases. Though this could possibly depend how exactly a plain+inline button would actually be used... wdyt?